selfdrive/sensord/sensors_qcom2.cc

```diff
@@ -88,6 +92,10 @@ void interrupt_loop(int fd, std::vector<Sensor *>& sensors, PubMaster& pm) {
       events.adoptWithCaveats(i, kj::mv(collected_events[i]));
     }
 
+    if (ts - init_ts < init_delay) {
+      continue;
+    }
```

Comment:
perhaps implement this as a generic data valid flag on the base sensor class. the default can be this 500ms init time, but we can then add other cases and override with more specific init checks in the future on a per-sensor basis

Reply:
I see, will change it

Reply:
I looked into it, I would like to move this to the sensor split event PR, its just way easier to implement there, in my mind its not worth the effort doing it here, due to the preallocation of sensorEvents and so on

Reply:
sounds good